dubbo-dependencies-bom/pom.xml

```diff
@@ -142,7 +142,7 @@
         <tomcat_embed_version>8.5.78</tomcat_embed_version>
         <jetcd_version>0.5.3</jetcd_version>
         <nacos_version>2.1.0</nacos_version>
-        <grpc.version>1.47.0</grpc.version>
+        <grpc.version>1.41.0</grpc.version>
```

Comment:
Why lower the grpc version?

Reply:
because the 1.47.0 version can't send ca request to envoy